tapir/settings.py

```diff
@@ -168,22 +167,25 @@
     EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
 elif EMAIL_ENV == "prod":
     EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
-    EMAIL_HOST = "smtp-relay.gmail.com"
-    EMAIL_HOST_USER = "mitglied@supercoop.de"
+    EMAIL_HOST = env("EMAIL_HOST", default="smtp-relay.gmail.com")
+    EMAIL_HOST_USER = env("EMAIL_HOST_USER", default="mitglied@supercoop.de")
     EMAIL_HOST_PASSWORD = env("EMAIL_HOST_PASSWORD")
     EMAIL_PORT = 587
     EMAIL_USE_TLS = True
 
-EMAIL_ADDRESS_MEMBER_OFFICE = "mitglied@supercoop.de"
-FROM_EMAIL_MEMBER_OFFICE = f"SuperCoop Mitgliederbüro <{EMAIL_ADDRESS_MEMBER_OFFICE}>"
+EMAIL_ADDRESS_MEMBER_OFFICE = env(
+    "EMAIL_ADDRESS_MEMBER_OFFICE", default="mitglied@supercoop.de"
+)
+COOP_NAME = env("COOP_NAME", default="SuperCoop Berlin")
```

Comment:
Interesting thoughts!
What makes COOP_NAME interesting is that you can merge changes (in particular i18n-related stuff in the .po files) a lot easier.
It could also make a docker-compose-only deployment possible, where you put your custom templates in volumes and define the coop name in env?
That might be too far fetched, though.

Reply:
Yes, I totally agree we should have `COOP_NAME` and many other vars! My suggestion would be to keep them in `settings.py` though and not set them through the environment.

My feeling is that new deployments only through the env aren't realistic, as every coop will probably require some small code change somewhere. I'm not even sure it's desirable: Having a git history of template and policy changes and building them all into the image is quite desirable I think. Also makes rollbacks easier in case something breaks, since almost all config is baked into the Docker image.